recipes/sqlpp11/all/conanfile.py

```diff
@@ -0,0 +1,28 @@
+from conans import ConanFile, tools
+import os
+
+
+class Sqlpp11Conan(ConanFile):
+    name = "sqlpp11"
+    license = "BSD 2-Clause License"
```

Comment:
As always, please use valid SPDX identifier https://spdx.org/licenses/

```suggestion
    license = "BSD-2-Clause"
```